evm/src/main/java/org/hyperledger/besu/evm/MainnetEVMs.java

```diff
@@ -948,6 +950,8 @@ public static void registerPragueOperations(
       final GasCalculator gasCalculator,
       final BigInteger chainID) {
     registerCancunOperations(registry, gasCalculator, chainID);
+    registry.put(new AuthOperation(gasCalculator));
```

Comment:
Ad a comment referring to the EIP these are for (like in registerCancunOperations)